server/server/fields.py

```diff
@@ -12,6 +12,8 @@
 from django.utils.html import mark_safe
 from django.utils.translation import gettext_lazy as _
 
+from phac_aspc.django.fields import CommaSeparatedCharField
+
 migration_ignored_attrs = [
     "help_text",
```

Comment:
Updated phac-helpers include all of the field subclasses, I might open up another PR later to just use those and we can get rid of this module (though we'd have to keep the CKEditor one)